RobotCode/snobot2018/src/org/snobot/elevator/IElevator.java

```diff
@@ -0,0 +1,26 @@
+package org.snobot.elevator;
+
+/**
+ * This is the interface for the elevator that allows someone to move the
+ * elevator up and down.
+ * 
+ * @author Josh
+ *
+ */
+public interface IElevator
```

Comment:
Need to add a story to have the elevator go to the different heights required for the competition.  These might be in addition to using the joystick to move up and down.  That needs to be discussed as part of the story.

Reply:
Agreed